specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/instancePools.json

```diff
@@ -0,0 +1,453 @@
+          },
+          "204": {
+            "description": "The specified instance pool does not exist."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Delete an instance pool": {
+            "$ref": "./examples/DeleteInstancePool.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "InstancePools"
+        ],
+        "description": "Updates an instance pool.",
+        "operationId": "InstancePools_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "name": "instancePoolName",
+            "in": "path",
+            "description": "The name of the instance pool to be updated.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The requested instance pool resource state.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/InstancePoolUpdate"
+            }
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated the instance pool.",
+            "schema": {
+              "$ref": "#/definitions/InstancePool"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidResourceRequestBody - The resource or resource properties in the request body is empty or invalid."
+          },
+          "202": {
+            "description": "Accepted"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Patch an instance pool": {
+            "$ref": "./examples/PatchInstancePool.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/instancePools": {
+      "get": {
+        "tags": [
+          "InstancePools"
+        ],
+        "description": "Gets a list of instance pools in the resource group",
+        "operationId": "InstancePools_ListByResourceGroup",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the list of instance pools.",
+            "schema": {
+              "$ref": "#/definitions/InstancePoolListResult"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***"
```

Comment:
Can this be expanded? It's not helpful in current form.

Reply:
Discussed we always return empty list if no instance pools found in resource group.
We do similar in https://github.com/Azure/azure-rest-api-specs/blob/master/specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/managedInstances.json#L45